### PR TITLE
In game console

### PR DESCRIPTION
**Pull request by jonathan2222**
Can add commands and bind them to functions.

### CODE REVIEW

File: LambdaEngine/Include/Game/ConsoleCommand.h [hunk 1]
```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+namespace LambdaEngine
+{
+	
+
+	struct LAMBDA_API Arg
+	{
+		enum EType { FLOAT, BOOL, INT, STRING, EMPTY };
```
Comment (Mumsfilibaba) [inline LambdaEngine/Include/Game/ConsoleCommand.h]: Is there a reason for this not being an enum class?

**Review by Mumsfilibaba — COMMENTED** [review]
Overall looks good. However two things:
1. Think it would be more readable if the class is encapsulated inside namespace LambdaEngine { }
instead of specifying LambdaEngine:: in front of everything.
2. The naming of members in the structs should start with capital letters.

File: LambdaEngine/Include/Game/ConsoleCommand.h (hunk 1 above)
Comment (jonathan2222) [reply LambdaEngine/Include/Game/ConsoleCommand.h]: no

**Review by Ceanze — APPROVED** [review]
Looks fine, previous comments addressed. Approved.

**Pull request merged**